project-koku/koku
# attempt to fix category filter/exclude logic

## Jira Ticket

[COST-3268](https://issues.redhat.com/browse/COST-3268)

## Description

This change will fix the filter and exclude logic for platform category

## Testing

1. Checkout Branch
2. Restart Koku
3. load OCP data with openshift- and kube- naming
4. hit http://localhost:8000/api/cost-management/v1/reports/openshift/costs/?group_by[project]=*&category=platform&exclude[project]=platform AND http://localhost:8000/api/cost-management/v1/reports/openshift/costs/?group_by[project]=*&category=platform&exclude[project]=platform&limit=30
5. Check both endpoints for platform category exclusion and inclusion

## Notes

...
